browser/components/BoardShowPage/CardModal.js

```diff
@@ -41,7 +41,37 @@ export default class CardModal extends Component {
     this.updateDescription(description)
   }
 
+  toggleLabel(){
+
+  }
+
   render(){
+    const Controls = ({card, closeModal, board, list}) => {
```

Comment:
Moved this here so we could pass in a function to labelCard bound to the component.